### PR TITLE
Updated timelines for pilot 3 resubmission. Closes #108 

### LINKED ISSUE #108: Pilot 3 Timeline update
@laxamanaj we Should update main WG website with our new timeline for resubmission, once that is done!

image

### PR DESCRIPTION
**Pull request by laxamanaj**
Closes #108

### CODE REVIEW

**Comment by laxamanaj** [comment]
Hi, @bms63 .  When you have the chance feel free to pull #108 branch and comment here if update looks ok (I couldn't add you as a reviewer on this PR).

@rpodcast , once @bms63 confirms this looks ok, can you merge for us?  Thanks in advance.

File: pilot3.qmd [hunk 1]
```diff
@@ -4,61 +4,64 @@ title: "R Submission Pilot 3"
...
 
-**Key evaluation aspects**: 
-  
-- For Submitter  
-  - Generation of a reproducible R environment with a proprietary R package from git and open source R packages using `{renv}`
-  - Preparation of R-based submission materials
-- For FDA Staff
-  - Reproduce analysis results by reproducing the R environment and retrieving both proprietary & open source packages
+-   For Submitter
+    -   Generation of a reproducible R environment with a proprietary R package from git and open source R packages using `{renv}`
```
Comment (bms63) [inline pilot3.qmd:13]: Do we need the from git?

**Review by bms63 — APPROVED** [review]
I'm loving it

File: pilot3.qmd (hunk 1 above)
Comment (laxamanaj) [reply pilot3.qmd:13]:
```suggestion
    -   Generation of a reproducible R environment installing open source R packages using `{renv}` and installing a proprietary R package following independent steps. In future, sponsor specific submissions, it is likely that sponsors may create their own internal proprietary R packages, which may not be publicly available. Thus, providing steps on how to submit and install a proprietary R package using independent steps allows others in the industry to follow the same approach.  In this working group we've tested these proprietary R package submission and installation steps succesfully : 
    - Pilot 1 - {pkglite}
    - Pilot 2 - {pkglite}
    - Pilot 3 - Github
    - Pilot 3 - Local install via .zip (Pending FDA feedback)

```

Comment (bms63) [reply pilot3.qmd:13]: oh i like this!!

**Pull request merged**